### DISCUSSION TITLE
What are you doing?

### DISCUSSION THREAD

**Discussion by eltikia** [eltikia/Perl-1.0#32] [Q&A] [answered]
Can you explain what has been done with the Perl 1 repository? I mean the recent commits made by @Stagyrite.

**Comment by Stagyrite** [comment, marked as answer]
The refreshed Perl 1 is supposed to run on modern operating systems. It wasn't even compiling at first. A non-compiling AWK to Perl translator was also taken care of. It now compiles on several modern operating systems, and that includes Windows (Cygwin). Multiple compiler warnings were eliminated.
